assets/src/activation-notice/app/components/successMessage.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import { useCallback } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { trackClick } from '../../../tracking';
+import { useConfig } from '../config';
+
+import Paragraph from './paragraph';
+import SecondaryButton from './link';
+
+const Message = styled.div`
+  min-width: 260px;
+  margin-right: 30px;
+  padding: 30px 0 30px;
+
+  ${SecondaryButton} {
+    @media ${({ theme }) => theme.breakpoint.tabletSmall} {
+      display: none;
+    }
+  }
+`;
+
+const Title = styled.h2`
+  font-family: ${({ theme }) => theme.fonts.title.family};
+  font-size: ${({ theme }) => theme.fonts.title.size};
+  line-height: ${({ theme }) => theme.fonts.title.lineHeight};
+  color: ${({ theme }) => theme.colors.primary};
+  margin: 0 0 10px;
+`;
+
+const PrimaryButton = styled.a`
```

Comment:
```suggestion
const PrimaryButton = styled.a.attrs({ role: 'button' })`
```

Reply:
Hmm I don't think it should have the button role.

It should just look like a button, but is an actual link, so adding a button role would only make it _less_ accessible.